frontend/src/pages/ApartmentPage.tsx

```diff
@@ -377,18 +402,23 @@ const ApartmentPage = ({ user, setUser }: Props): ReactElement => {
     setReviewOpen(true);
   };
 
+  const handleMapModalClose = () => {
+    setMapOpen(false);
+    setMapToggle((prev) => !prev);
+  };
+
   const Modals = landlordData && apt && (
     <>
       <MapModal
         aptName={apt!.name}
         open={mapOpen}
-        onClose={() => setMapOpen(false)}
+        onClose={handleMapModalClose}
         setOpen={setMapOpen}
```

Comment:
Your setOpen parameter is no longer being used, so maybe remove setOpen as a parameter?